Yoast/wordpress-seo
# Remove jed

Based on https://github.com/Yoast/wordpress-seo/pull/17075
## Context

We have a complicated set of build steps that help us generate translation files. translate.wordpress.org now supports for automatic generation of translation files, so we no longer need our own pipeline. This only works for calls to `wp.i18n.__` and such which the plugin's text-domain: `"wordpress-seo"`.

## Summary

This PR can be summarized in the following changelog entry:

* Make use of WordPress' internationalisation package in javascript.
* Change the translation text domain to `wordpress-seo` in all javascript translations.

## Relevant technical choices:

* I chose to use the `@wordpress/babel-plugin-makepot` plugin to generate pot files for yoastseo js. This is consistent with the other packages.
* I've been told that translate.wp.org doesn't play well with concatenated strings in `__` calls. So, long strings are now on a single line with an eslint-ignore-next-line comment above it.

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* Run `grunt build:i18n` and `grunt build:js` 
* Change your site language to anything other than english.
* Verify that the entire plugin is still translated as it was before. Check at least:
    * The content analysis
    * The configuration wizard
    * The settings pages
    * The snippet preview+editor
    * The schema blocks, like the Yoast How-to block.


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

* Change your site language to anything other than english.
* Verify that the entire plugin is still translated as it was in the previous release. Check at least:
    * The content analysis
    * The configuration wizard
    * The settings pages
    * The snippet preview+editor
    * The schema blocks, like the Yoast How-to block

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

* Javascript translations

## UI changes


## Other environments


## Documentation


## Quality assurance


Fixes FIX-1
